src/renderers.ts

```diff
@@ -1,17 +1,22 @@
-import { BooleanControlRegistryEntry } from "./controls/BooleanControlRegistryEntry";
-import { LabelRendererRegistryEntry } from "./controls/LabelRendererRegistryEntry";
-import { TextControlRegistryEntry } from "./stories/controls/TextControlRegistryEntry";
-import {
-  UnknownCellRegistryEntry,
-  UnknownRendererRegistryEntry,
-} from "./controls/UnknownControlRegistryEntry";
-import { VerticalLayoutRendererRegistryEntry } from "./layouts/VerticalLayoutRegistryEntry";
+import { JsonFormsRendererRegistryEntry,JsonFormsCellRendererRegistryEntry, isBooleanControl, isStringControl, rankWith, uiTypeIs } from "@jsonforms/core";
+import { withJsonFormsControlProps, withJsonFormsLabelProps, withJsonFormsCellProps, withJsonFormsLayoutProps } from "@jsonforms/react";
 
-export const rendererRegistryEntries = [
-  BooleanControlRegistryEntry,
-  LabelRendererRegistryEntry,
-  TextControlRegistryEntry,
-  UnknownRendererRegistryEntry,
-  VerticalLayoutRendererRegistryEntry,
+import { BooleanControl } from "./controls/BooleanControl";
+import { AlertControl } from "./controls/AlertControl";
+import { TextControl } from "./controls/TextControl";
+import { UnknownControl } from "./controls/UnknownControl";
+import { VerticalLayoutRenderer } from "./layouts/VerticalLayout";
+
+
+// Ordered from lowest rank to highest rank. Higher rank renderers will be preferred over lower rank renderers.
+export const rendererRegistryEntries: JsonFormsRendererRegistryEntry[] = [
```

Comment:
I think naming the renderers is a good thing/opportunity to make the code more readable. What do you think? Are we missing out on anything by omitting renderer-specific names?

Reply:
Users of this package would typically import this variable and then pass it directly to `JsonForms.renderers`. Is passing all renderers what they want to do? Internally I've seen us do something like:
```
renderers={[...rendererRegistryEntries, ...(customRendererRegistryEntries ?? [])]}
```
This could cause conflicts if we are overriding one of these renderers with a custom one. I think what I'm contemplating is whether we want to give users the ability to import individual registry entries to build their own array like this.

Reply:
I think the naming convention depends on some other design choices we need to make. Like, does it make sense to have renderer registry entries for each field display type and pull branching logic up out of controls, so you get renderer names like MultiLineInputRenderer, SingleLineInputRenderer, PasswordInputRenderer (each of which is currently handled by the TextControl/TextRenderer), or do we keep that display branching logic at the Control level and use more general (and fewer) Renderer names.

I don't think it blocks this pr, your reorganization of the code is great!